embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java

```diff
@@ -201,17 +202,19 @@ private void setHttpProxyInAwsClient(ClientConfiguration clientConfig, HttpProxy
 
     private FileList listFiles(PluginTask task)
     {
+        LOGGER.info("Start listing file with prefix [{}]", task.getPathPrefix());
```

Comment:
Here I am being picky, should this be a "warn" instead? Other than that, LGTM.

Reply:
I think we are providing information here, not warning user about anything.

Reply:
Oops, silly me, I mean this line https://github.com/embulk/embulk-input-s3/pull/68/files/9398acec7e3391f01d483ef02d5996d13a42d896#diff-6bf6730afed89d21ba77d97065bace32R259.